sgm/models/diffusion.py

```diff
@@ -245,9 +250,9 @@ def sample(
     ):
         randn = torch.randn(batch_size, *shape).to(self.device)
 
-        denoiser = lambda input, sigma, c: self.denoiser(
-            self.model, input, sigma, c, **kwargs
-        )
+        def denoiser(input, sigma, c):
+            return self.denoiser(self.model, input, sigma, c, **kwargs)
```

Comment:
Don't use a `lambda` where a `def` will do. (That said, this is really just `partial()`...)